include/EVT/utils/time.hpp

```diff
@@ -20,5 +20,35 @@ namespace EVT::core::time
      * @param ms The number of milliseconds to wait for
      */
     void wait(uint32_t ms);
+
+    /**
+     * Struct representing a timestamp.
+     */
+    struct TimeStamp {
+        /**
+         * The current year
+         */
+        uint16_t year;
+        /**
+         * The current month (0-12)
```

Comment:
Shouldn't this be (1-12)? Or (0-11)?

Reply:
Lol yes, good catch, 1-12. I'll fix that now